modular_skyrat/modules/modular_clothing/armor/armor.dm

```diff
@@ -0,0 +1,72 @@
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
+	icon_state = "hardsuit-secexo"
+	inhand_icon_state = "hardsuit-secexo"
+	max_integrity = 250
+	armor = list("melee" = 30, "bullet" = 30, "laser" = 30, "energy" = 10, "bomb" = 25, "bio" = 50, "rad" = 0, "fire" = 55, "acid" = 70)
+	allowed = list(/obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/flashlight, /obj/item/gun/ballistic, /obj/item/gun/energy, /obj/item/kitchen/knife/combat, /obj/item/melee/baton, /obj/item/melee/classic_baton/telescopic, /obj/item/reagent_containers/spray/pepper, /obj/item/restraints/handcuffs, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman) //I had to do this all snowflake style because it just would not accept any sort of global list, fucking kill me
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/security_armor
+	clothing_flags = THICKMATERIAL
+	mutant_variants = STYLE_DIGITIGRADE
+	slowdown = 0
+	flags_inv = NONE
+/*Let me just put a big fucking note right here because I know somebody is going to complain about this:
+No, this does not let you stack the helmet-given flash resistance and Nightvision or Thermal glasses with no consequence.
+Sure, you can do it 100%, but it will invalidate the helmet-given flash protection and render you vulnerable
+just as if you were wearing them separate. Stop screaming to me about this. Please.*/
+
+/obj/item/clothing/suit/space/hardsuit/security_armor/Initialize()
+	. = ..()
+	if(!allowed)
+		allowed = GLOB.security_vest_allowed
+
+/obj/item/clothing/head/helmet/space/hardsuit/security_armor/equipped(mob/living/carbon/human/user, slot)
+	..()
+	if (slot == ITEM_SLOT_HEAD)
+		var/datum/atom_hud/DHUD = GLOB.huds[DATA_HUD_SECURITY_ADVANCED]
+		DHUD.add_hud_to(user)
+
+/obj/item/clothing/head/helmet/space/hardsuit/security_armor/dropped(mob/living/carbon/human/user)
+	..()
+	if (user.head == src)
+		var/datum/atom_hud/DHUD = GLOB.huds[DATA_HUD_SECURITY_ADVANCED]
+		DHUD.remove_hud_from(user)
+
+/obj/item/clothing/head/helmet/space/hardsuit/security_armor/hos
+	name = "head of security's techhelmet"
+	desc = "A specialized exoskeleton armor helmet built into a suit of armor; offers decent protection, and comes with a flash-resistant HUD visor and headlamp."
+	icon_state = "hardsuit0-hosexo"
+	inhand_icon_state = "hardsuit0-hosexo"
+	hardsuit_type = "hosexo"
+	mutant_variants = STYLE_MUZZLE
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR
+
+/obj/item/clothing/suit/space/hardsuit/security_armor/hos
+	name = "head of security's techarmor"
+	desc = "A specialized exoskeleton armor suit comprised of flexible protective shielding. This particular suit has been designed specifically for the station security commander."
+	icon_state = "hardsuit-hosexo"
+	inhand_icon_state = "hardsuit-hosexo"
+	max_integrity = 300
+	armor = list("melee" = 35, "bullet" = 30, "laser" = 30, "energy" = 10, "bomb" = 25, "bio" = 50, "rad" = 0, "fire" = 70, "acid" = 90)
```

Comment:
The appropriate values for the HoS armor is as follows and should be shared by the helmet.

```armor = list(MELEE = 30, BULLET = 30, LASER = 30, ENERGY = 40, BOMB = 25, BIO = 100, RAD = 0, FIRE = 70, ACID = 90, WOUND = 10)```

This is just taking the existing greatcoats values and adding 100 bio armor. It's actually that resilient. The same cannot be said of his beret but that seems to be an oversight since there was a balance pass on armor and helmets being mismatched for values. Also, the greatcoat seemingly has less melee armor than its armor vest counterpart. That is a bit weird. I'd recommend just bringing it up to 35 as well.

This value should obviously be shared by the helmet as well.

Reply:
Values were from the old base and are supposed to be shared between both sets; fixing now.